doc/input-output-reference/src/overview/group-unitary-equipment.tex

```diff
@@ -227,7 +227,7 @@ \subsubsection{Inputs}\label{inputs-049}
 
 \paragraph{Field: Cooling Supply Air Flow Rate Method}\label{field-cooling-supply-air-flow-rate-method-000}
 
-This alpha field defines the supply air flow method during cooling operation. Available choices are SupplyAirFlowRate, FlowPerFloorArea, FractionOfAutosizedCoolingValue, FlowPerCoolingCapacity. For each of the choices, a corresponding air flow rate for cooling must be specified.
+This alpha field defines the supply air flow method during cooling operation. Available choices are SupplyAirFlowRate, FlowPerFloorArea, FractionOfAutosizedCoolingValue, FlowPerCoolingCapacity. For each of the choices, a corresponding air flow rate for cooling must be specified. If the system does not have a heating coil a 0 may be entered for heating air flow rate and/or no load supply air flow rate to turn the fan off when cooling is not required.
 
```

Comment:
Shouldn't this one talk about the cooling coil, not the heating coi?

Reply:
I was trying to talk about the other coil here. I guess that does sound confusing. I'll flip it to read like this in both places:

If the system does not have a cooling coil a 0 may be entered for cooling air flow rate and/or no load supply air flow rate to turn the fan off when cooling is not required.

Reply:
"Must be greater than 0 if the xxxxing coil is present"